client/settings/general-settings/style.scss

```
@@ -0,0 +1,37 @@
+.general-settings {
+	&__bank-information {
+		display: flex;
+		flex-direction: column;
+
+		:first-child {
+			flex-grow: 1;
+		}
+
+		@include breakpoint( '>800px' ) {
+			align-items: center;
+			justify-content: space-between;
+			flex-direction: row;
+			flex-wrap: nowrap;
+		}
+	}
+
+	&__account-statement-wrapper {
+		position: relative;
+
+		.input-help-text {
+			position: absolute;
+			right: 10px;
+			top: 12px;
```

```suggestion
			top: 11px;
```

I think this should be `11` because when I do this:

```css
top: 20px; /* = height of input field / 2 */
transform: translateY( -50% );
```

The computed transform says:

```
matrix(1, 0, 0, 1, 0, -9.1)
```

So (skipping the `.1`): `20 - 9 = 11`.